src/identity/platform.rs

```diff
@@ -0,0 +1,96 @@
+//! Kernel cmdline parsing - utility functions
+//!
+//! NOTE(lucab): this is not a complete/correct cmdline parser, as it implements
+//!  just enough logic to extract the platform ID value. In particular, it does not
+//!  handle separator quoting/escaping, list of values, and merging of repeated
+//!  flags.
```

Comment:
Probably worth mentioning that this code comes from Afterburn? Just so that any bugs found here should probably also be applied there.

Reply:
Ack, I've added a note and direct link to Afterburn source.